### DISCUSSION TITLE
What plugin(s) would be loaded *only* under VSCode?

### DISCUSSION THREAD

**Discussion by proofer** [vscode-neovim/vscode-neovim#2314] [Q&A] [answered]
Having reviewed the wiki here for the technique of conditionally loading plugins using lazy.nvim, I'm wondering what what plugin(s) would be loaded *only* under VSCode?

**Comment by justinmk** [comment]
That is controlled by you (your config)

**Reply by proofer** [reply]
Yes, sorry, I wasn't clear. I understand that. What I'm curious about is what plugin(s) would be example(s) of those which someone would want to load only under VSC. I can't think of any, but that could be just ignorance on my part.

**Reply by justinmk** [reply, marked as answer]
Any keybindings that you define in `init.lua` only for vscode would be a vscode-only "plugin".

vscode-neovim itself ships with some "vscode only" plugins. But externally, I don't know of any vscode-only Nvim plugins and would actually discourage that.
